website/content/docs/configuration/time-format.mdx

```diff
@@ -0,0 +1,39 @@
+---
+layout: docs
+page_title: Time String Format
+description: A description of Vault's time string format used throughout its configuration.
+---
+
+# Time String Format
+
+Vault uses a unique but ultimately simple time string format as part of its configuration and APIs.
+This format is used throughout Vault wherever time units, such as durations, need to be given. For
```

Comment:
suggestion: replace 'need to be given' with 'are provided'. Or, you can also say 'are given'.